lighthouse-core/scripts/update-flow-fixtures.js

```diff
@@ -7,42 +7,74 @@
 
 import fs from 'fs';
 
-import open from 'open';
 import puppeteer from 'puppeteer';
 
 import {LH_ROOT} from '../../root.js';
 import UserFlow from '../fraggle-rock/user-flow.js';
+import config from '../fraggle-rock/config/default-config.js';
 
-(async () => {
-  const browser = await puppeteer.launch({headless: false});
+/**
+ * @param {import('puppeteer').Browser} browser
+ */
+async function buildSampleFlowResult(browser) {
+  const page = await browser.newPage();
+  const flow = new UserFlow(page);
 
-  try {
-    const page = await browser.newPage();
-    const flow = new UserFlow(page);
+  await flow.navigate('https://www.mikescerealshack.co');
 
-    await flow.navigate('https://www.mikescerealshack.co');
+  await flow.startTimespan({stepName: 'Search input'});
+  await page.type('input', 'call of duty');
+  const networkQuietPromise = page.waitForNavigation({waitUntil: ['networkidle0']});
+  await page.click('button[type=submit]');
+  await networkQuietPromise;
+  await flow.endTimespan();
 
-    await flow.startTimespan({stepName: 'Search input'});
-    await page.type('input', 'call of duty');
-    const networkQuietPromise = page.waitForNavigation({waitUntil: ['networkidle0']});
-    await page.click('button[type=submit]');
-    await networkQuietPromise;
-    await flow.endTimespan();
+  await flow.snapshot({stepName: 'Search results'});
 
-    await flow.snapshot({stepName: 'Search results'});
+  await flow.navigate('https://www.mikescerealshack.co/corrections');
 
-    await flow.navigate('https://www.mikescerealshack.co/corrections');
+  const flowResult = flow.getFlowResult();
+
+  fs.writeFileSync(
+    `${LH_ROOT}/lighthouse-core/test/fixtures/fraggle-rock/reports/sample-flow-result.json`,
+    JSON.stringify(flowResult, null, 2)
+  );
+}
+
+/**
+ * @param {import('puppeteer').Browser} browser
+ */
+async function buildLocalizedFlowResult(browser) {
```

Comment:
same here, let's use swapLocale instead? and/or create a flow-specific one

Reply:
I think I'll just revert this one